src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/altar/AncientAltar.java

```diff
@@ -28,28 +30,27 @@
  */
 public class AncientAltar extends SlimefunItem {
 
-    private final int speed;
+    private static final int DEFAULT_STEP_DELAY = 8;
```

Comment:
javadocs on this would be nice.
Can also add yourself as an `@author` to this class.